index.html

```diff
@@ -1,32 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang = en>
     <head>
-        <title>Em Suarez</title>
-        <meta name="viewport" content=""width=device-width, initial-scale="1.0">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width", initial-scale="1.0">
```

Comment:
```suggestion
        <meta name="viewport" content="width=device-width, initial-scale=1">
```
The value for `content` is _all_ of `width=device-width, initial-scale=1`... Unless you're trying to do something special here, best just to keep it exactly like the example you're following.